src/main/java/com/microfocus/application/automation/tools/octane/model/SonarHelper.java

```diff
@@ -18,9 +18,8 @@
  * ___________________________________________________________________
  */
 
-package com.microfocus.application.automation.tools.model;
+package com.microfocus.application.automation.tools.octane.model;
```

Comment:
If this class was serialized and introduced in previous versions, this change might break their configurations.

Reply:
This class is new for this version.

Reply:
... and it is NOT serializable as well.